buildSrc/subprojects/jvm/src/main/kotlin/gradlebuild/unittest-and-compile.gradle.kts

```diff
@@ -43,6 +43,7 @@ configureCompile()
 configureSourcesVariant()
 configureJarTasks()
 configureTests()
+fixCacheMisses()
```

Comment:
:o: I would give this a different name, since it does not fix the miss directly. Like:
`failOnEmptySourceFolder`

Reply:
Done!